scripts/EN/Harry

```diff
@@ -1857,32 +1857,32 @@ DrawBackground(); // 0x2080: 00000065
 PlayMusic(0x67, 0x1); // 0x2084: 0000206c
 LoadSecondEffect(0xd, 0x1, 0x0); // 0x208c: 0000306e
 LoadSecondEffect(0x5f, 0x2, 0x1); // 0x2096: 0000306e
-ShowText((1084)"¥"Thank you, but...I'm no good with guns. I haven't even touched one. I can't handle it.¥""); // 0x20a0: 0000100c
+ShowText((1084)"¥"Thank you, but…W2I'm no good with guns.¥nW2I haven't even touched one.¥nW2I can't handle it.¥""); // 0x20a0: 0000100c
 HandleInput(); // 0x20a6: 00000010
 LoadSecondEffect(0x0, 0x0, 0x1); // 0x20aa: 0000306e
 CleanScreen(); // 0x20b4: 00000064
 LoadBackground(0x18); // 0x20b8: 00001069
 LoadEffect(0x0, 0x0, 0x0); // 0x20be: 0000306d
 DrawBackground(); // 0x20c8: 00000065
-ShowText((1085)"Amazed, Cybil takes a deep breath. ¥"It's simple. I'll show you how to use it. See, first hold it in your right hand, then place your forefinger on the trigger.¥" I reluctantly followed her instructions."); // 0x20cc: 0000100c
+ShowText((1085)"Amazed, Cybil takes a deep breath. W1¥"It's simple. I'll show you how to use it. W2See, first hold it in your right hand, W1then place your index finger on the trigger.¥"¥nW2I reluctantly followed her instructions."); // 0x20cc: 0000100c
```

Comment:
Although forefinger is not wrong, I have changed it to "index finger" because I think that sounds a lot better. No idea why I didn't do this originally!